src/core/Akka.API.Tests/CoreAPISpec.ApproveRemote.approved.txt

```diff
@@ -1309,6 +356,962 @@ namespace Akka.Remote
         public override string ToString() { }
     }
 }
+namespace Akka.Remote.Proto
+{
+    public sealed class AckAndEnvelopeContainer : Google.ProtocolBuffers.GeneratedMessage<Akka.Remote.Proto.AckAndEnvelopeContainer, Akka.Remote.Proto.AckAndEnvelopeContainer.Builder>
+    {
```

Comment:
This part is current code. 
As you can see, its still the same. Just wrapped in an namespace.

However because if the difference in layout. The entire file became hard to compare. Which is also why the api approval changes are huge.